.appveyor.yml

```diff
@@ -68,11 +68,13 @@ before_build:
   - '%APPVEYOR_BUILD_FOLDER%\testing\dependencies\appveyor\extract.bat'
 
 build_script:
+  - bash -c "cd /c/projects/cmake-cookbook-no-symlinks && pipenv lock"
```

Comment:
Out of curiosity, why lock first? Because otherwise `install` times out? Have you tried updating the `Pipfile.lock` locally and then committing that?

Reply:
Perhaps I should find a comment. The reason was that after upgrade to Python 3.7, Pipenv complained about incorrect checksums. This is strange and unexpected and after some poking I "solved" it by locking the versions and thus ignoring conflicting hashes. Nicer solutions are welcome. Also I would like to understand why but I also needed to move on :-)

Reply:
As long as we understand why it's needed :smile: